Source/Diagnostics/ReducedDiags/ParticleHistogram2D.cpp

```diff
@@ -298,7 +298,7 @@ void ParticleHistogram2D::WriteToFile (int step) const
     data.setPosition<amrex::Real>({0.5, 0.5});
 
     auto dataset = io::Dataset(
-            io::determineDatatype<double>(),
+            io::determineDatatype<amrex::Real>(),
```

Comment:
Wouldn't the properties be in `ParticleReal`?
```suggestion
            io::determineDatatype<amrex::ParticleReal>(),
```

Reply:
Yes, you are right!

Reply:
However, we also have to modify the datatype of `m_h_data_2D` to be coherent (I've done that in my latest commit):  `amrex::TableData<amrex::Real,2> m_h_data_2D` ->   `amrex::TableData<amrex::ParticleReal,2> m_h_data_2D`